opl/opl_sdl.c

```diff
@@ -381,7 +250,7 @@ static int OPL_SDL_Init(unsigned int port_base)
     callback_mutex = SDL_CreateMutex();
     callback_queue_mutex = SDL_CreateMutex();
 
-    Mix_HookMusic(OPL_Mix_Callback, NULL);
+    I_OAL_HookMusic(OPL_Callback);
```

Comment:
So, the OPL emulstor hoocks into OAL and this hooks into SDL? Or how do I have to understand this?

Reply:
No, we don't hook into SDL anymore, I removed SDL_Mixer from this PR completely. It's just a file called `opl_sdl.c` because we're still using a lot of SDL functions in it (not audio).